src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
+@import './utils/variables';
+
 body {
   margin: 0;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  border: 2px solid red;
```

Comment:
Inline styles should be avoided according to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). Consider removing the border style from the `body` element or moving it to a separate debugging stylesheet if it's only used for development purposes.